Base/usr/share/man/man1/crash.md

```diff
@@ -15,6 +15,10 @@ and can be used to simulate many different kinds of crashes.
 
 Some crash tests are only available on certain architectures.
 
+Some crash tests are excluded from the `-A` test, since depending on the hardware or implementation they may or may not crash.
+- Priviledged instructions in user mode are permitted by QEMU on some architectures such as x86. Therefore, this crash may not fail. See [discussion on pull request 10042](https://github.com/SerenityOS/serenity/pull/10042#issuecomment-920408568).
+- Misaligned instruction fetches are not required to trap on RISC-V, and they indeed do not trap in QEMU.
```

Comment:
They can also be caught and handled by the firmware (e.g. OpenSBI does that).

Reply:
Oops, misaligned instruction fetches are never caught.
Loads/stores are though.

Reply:
That's why I am only testing misaligned instructions for now, since it seems more likely to me that the hardware would have an issue with them.